lib/mindwendel/accounts.ex

```diff
@@ -111,4 +112,9 @@ defmodule Mindwendel.Accounts do
       {:ok, _} -> merge_brainstorming_user(brainstorming, get_or_create_user(user_id))
     end
   end
+
+  def delete_inactive_users(after_days \\ 30) do
+    date_time = Timex.now() |> Timex.shift(days: -1 * after_days)
+    Repo.delete_all(from u in User, where: u.updated_at < ^date_time)
```

Comment:
This delete filter does not consider users that have added or labelled an idea yesterday because this delete filter only looks at the field `updated_at` of the user. So this means that we might be deleting active users.

Given that users are bound to the session, we might consider setting the `max_age` of the session to the `after_save` param, see:
- https://hexdocs.pm/plug/Plug.Conn.html#put_resp_cookie/4
- https://stackoverflow.com/questions/42696744/how-to-make-a-session-last-longer

Whatever we do, we should add a comment to document our thinking and conclusion.

Reply:
sounds like a good solution with the max age...but this is then dependant on your other max_age pr.